src/router.js

```diff
@@ -0,0 +1,44 @@
+const {
+  task1,
+  task2,
+  task3,
+  setDataGlobal,
+  writeDataInFile,
+} = require('./controller');
+
+function notFound(res) {
+  res.setHeader('Content-Type', 'application/json');
+  res.statusCode = 404;
+  res.write('404');
+  res.end();
```

Comment:
This is tricky right from the example. It has two drawbacks:
First of all this code sets JSON header but the response has invalid JSON format. `'404'` - is not a JSON
Second drawback is `res.write` and `res.end`. These two lines can be replaced by one line `res.end('404');`